key/src/test/java/software/amazon/kms/key/UpdateHandlerTest.java

```diff
@@ -55,25 +55,63 @@ public class UpdateHandlerTest {
             .keyUsage(KeyUsageType.ENCRYPT_DECRYPT.toString())
             .origin(OriginType.AWS_KMS.toString())
             .keySpec(KeySpec.SYMMETRIC_DEFAULT.toString())
-            .keyPolicy(TestConstants.DESERIALIZED_KEY_POLICY)
+            .keyPolicy(TestConstants.KEY_POLICY)
+            .tags(ImmutableSet.of(Tag.builder()
+                .key("Key")
+                .value("Value")
+                .build()));
+    private static final ResourceModel.ResourceModelBuilder KEY_MODEL_NO_POLICY_BUILDER =
+        ResourceModel.builder()
+            .keyId("mock-key-id")
+            .arn("mock-arn")
+            .description("mock-description")
+            .enabled(true)
+            .enableKeyRotation(false)
+            .multiRegion(false)
+            .keyUsage(KeyUsageType.ENCRYPT_DECRYPT.toString())
+            .keySpec(KeySpec.SYMMETRIC_DEFAULT.toString())
+            .tags(ImmutableSet.of(Tag.builder()
+                .key("Key")
+                .value("Value")
+                .build()));
+    private static final ResourceModel.ResourceModelBuilder KEY_MODEL_DEFAULT_POLICY_BUILDER =
+        ResourceModel.builder()
+            .keyId("mock-key-id")
+            .arn("mock-arn")
+            .description("mock-description")
+            .enabled(true)
+            .enableKeyRotation(false)
+            .multiRegion(false)
+            .origin(TestConstants.KEY_ORIGIN)
```

Comment:
Nit: You can directly define OriginType.AWS_KMS.toString();

Reply:
I would disagree, Lets use TestConstants whenever possible so we can keep things in one place.

Reply:
Then we should also change for the rest whenever possible so that everything looks similar